.eslintrc.json

```diff
@@ -0,0 +1,35 @@
+{
+  "parser": "babel-eslint",
+  "extends": "airbnb",
```

Comment:
_Just_ realized this is `linter`, so it might actually be using React :stuck_out_tongue:.